devops/service-endpoints.md

```diff
@@ -0,0 +1,26 @@
+# 3Box service endpoints
+
+3Box provides a set of services that makes the 3Box network more resilient. Here all of the service endpoints are documented. To see APIs and documentation please refer to the repo for the given service.
```

Comment:
Update first sentence to: "3Box provides a set of services that make the 3Box network more resilient, convenient, scalable, and performant."